packages/web/app/src/components/ui/tabs.tsx

```diff
@@ -6,6 +6,21 @@ import * as TabsPrimitive from '@radix-ui/react-tabs';
 
 const Tabs = TabsPrimitive.Root;
 
+const TabsListMenu = React.forwardRef<
```

Comment:
```suggestion
const TabsList = React.forwardRef<
```
can it be similar to radix?

Reply:
maybe use `variant="..."` instead since there is 2 types of tabs